internal/dlc/fundtx_test.go

```diff
@@ -18,17 +21,48 @@ var testTxID = "14a0810ac680a3eb3f82edc878cea25ec41d6b790744e5daeef"
 // setup mocke wallet
 func setupTestWallet() *mocks.Wallet {
 	w := &mocks.Wallet{}
-	_, pub := test.RandKeys()
+	priv, pub := test.RandKeys()
 	w.On("NewPubkey").Return(pub, nil)
+	w = mockWitnessSignature(w, pub, priv)
 	return w
 }
 
-func newTestUtxos(amt btcutil.Amount) []wallet.Utxo {
+func mockSelectUnspent(
+	w *mocks.Wallet, balance, change btcutil.Amount, err error) *mocks.Wallet {
 	utxo := wallet.Utxo{
 		TxID:   testTxID,
-		Amount: float64(amt) / btcutil.SatoshiPerBitcoin,
+		Amount: float64(balance) / btcutil.SatoshiPerBitcoin,
 	}
-	return []wallet.Utxo{utxo}
+	w.On("SelectUnspent",
+		mock.Anything, mock.Anything, mock.Anything,
+	).Return([]wallet.Utxo{utxo}, change, err)
+
+	return w
+}
+
+func mockWitnessSignature(
```

Comment:
mock wallet mocks `WitnessSignature` to return sign created by using a given private key.